petab/visualize/helper_functions.py

```diff
@@ -438,6 +438,9 @@ def get_data_to_plot(vis_spec: pd.DataFrame,
         # check correct observable
         bool_observable = (m_data.observableParameters[ind_meas[0]] ==
                            m_data.observableParameters)
+        # special handling, if column in m_data.observableParameters is empty
+        if np.isnan(m_data.observableParameters[ind_meas[0]]):
+            bool_observable = (np.isnan(m_data.observableParameters))
```

Comment:
Superfluous parentheses

Reply:
one could somehow add a test to tests/test_visualization.py just showing that the plots are not empty even when the column is empty.

Reply:
The actual plotting makes tests so slow. Better somehow check before, that there is meaningful data to plot. Otherwise agreed.

Reply:
yes. not sure if it helps to boil down the plotting test cases (less data, less conditions).

Reply:
I would be very much in favor of actually using unit tests instead of full examples. However, this should be part of another pull-request.

Reply:
so, what is the plan now for the tests...? Shall I add a new test, or is it fine without...?

Reply:
For me it's enough to address it with #165 , which will probably change some more things here anyways.

Reply:
yes yes, if you tested it "somehow", i.e. verified that for some of your models this works now, that's fine. the tests will need to be rewritten substantially, so that's too much work now.